src/lib.rs

```diff
@@ -95,7 +90,9 @@ mod tests {
                 &mut mime,
             );
 
-            assert_eq!(res, 0);
+            assert_eq!(res, MAIL_NO_ERROR as libc::c_int);
```

Comment:
Maybe we should change the api to actually return `libc::c_uint` so it matches the error types? (Or change the error types to `libc::c_int`).